src/main/java/uk/gov/hmcts/reform/juddata/camel/route/ParentOrchestrationRoute.java

```diff
@@ -77,11 +77,11 @@ public void startRoute() throws Exception {
         String parentName = ROUTE + "." + parentRouteName;
         String childNames = ROUTE + "." + parentRouteName + "." + CHILD_ROUTES;
 
-        List<String> childrenList = environment.containsProperty(childNames)
+        List<String> dependents = environment.containsProperty(childNames)
```

Comment:
dependant spelling is wrong.. and i think children is still a better choice of word than dependant or if you want to use dependant than perhaps better choice is "dependantRoutes"

Reply:
done